imports/plugins/core/orders/client/components/OrderPreviousRefunds.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import PropTypes from "prop-types";
+import moment from "moment";
+import withStyles from "@material-ui/core/styles/withStyles";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import CardHeader from "@material-ui/core/CardHeader";
+import Divider from "@material-ui/core/Divider";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import { i18next } from "/client/api";
+
+const styles = (theme) => ({
+  dividerSpacing: {
+    marginBottom: theme.spacing(4),
+    marginTop: theme.spacing(4)
+  },
+  fontWeightSemiBold: {
+    fontWeight: theme.typography.fontWeightSemiBold
+  }
+});
+
+/**
+ * @name OrderPreviousRefunds
+ * @param {Object} props Component props
+ * @returns {React.Component} returns a React component
+ */
+function OrderPreviousRefunds(props) {
+  const { classes, order } = props;
+  const { refunds: unsortedRefunds } = order;
+  // sort refunds by date, newest first, for display purposes
+  const refunds = unsortedRefunds.slice().sort((refundA, refundB) => new Date(refundB.createdAt) - new Date(refundA.createdAt));
```

Comment:
It could wait for a future PR, but I'd prefer to see this implemented through a `sort` param on the server resolver.

Reply:
I was debating doing this on server vs. client. In the end, I decided client because I felt like the client should take in the raw data however it came and decide how to sort it on it's own, independent.

I guess we could do it in both places, if the client wanted to re-sort it differently than the server was sending it, but was thinking one re-sort is less manipulation than two and to do it this way.

Reply:
https://github.com/reactioncommerce/reaction/issues/5418